servicecontrol/capacity-and-planning.md

```diff
@@ -61,4 +61,4 @@ The transports supported by ServiceControl out-of-the-box (i.e. MSMQ, RabbitMQ,
 
 Azure Queues and Service Bus throughput varies significantly based on deployment options and multiple related variables inherent to cloud deployment scenarios.
 
-It is recommended that you plan and perform realistic throughput tests on ServiceControl using the transport of your choice and deployment options that are as close as possible to your planned production deployment. For additional questions or information please [contact Particular Software](http://particular.net/contactus).
\ No newline at end of file
+The recommended approach is that customers plan and perform realistic throughput tests on ServiceControl using the transport of your choice and deployment options that are as close as possible to your planned production deployment. For additional questions or information please [contact Particular Software](http://particular.net/contactus).
```

Comment:
Doesn't read well with the you being changed to Customers, and still having fragments like transport of your choice, etc.

How about:
The recommended approach is that customers plan and perform realistic throughput tests on ServiceControl using the transport of their choice and select deployment options that are as close as possible to their production environment. For additional questions or information please [contact Particular Software](http://particular.net/contactus).